README.md

```diff
@@ -1,5 +1,11 @@
-# ppa
-A PPA repository for SecureDNA packages
+# PPA
+A PPA repository for SecureDNA packages.  Currently supports [Ubuntu amd64](https://github.com/SecureDNA/ppa).
+
+SecureDNA is a free, non-profit screening platform designed to safeguard DNA synthesis everywhere.
+
+- You can find information about the project as a whole, including papers and a live demo, at the [main site](https://securedna.org).
+- Source for the client and servers is available in our [monorepo](https://github.com/SecureDNA/SecureDNA).
+- To use the production servers with real databases, you'll need to [obtain certificates](https://securedna.org/start/), but you can compile the entire system below and run test servers with test databases and your own certificates.
```

Comment:
I don't think of the PPA as involving compilation. Also, does it include the servers?

Reply:
I should maybe read what I copy from other repos. :)  Now I've rewritten a bunch of it for clarity; tnx.